sljit_src/sljitNativeLOONGARCH_64.c

```diff
@@ -2934,7 +2932,7 @@ SLJIT_API_FUNC_ATTRIBUTE sljit_s32 sljit_emit_op_flags(struct sljit_compiler *co
 			break;
 		case SLJIT_ATOMIC_STORED:
 		case SLJIT_ATOMIC_NOT_STORED:
-			FAIL_IF(push_inst(compiler, SLTUI | RD(dst_r) | RJ(EQUAL_FLAG) | IMM_I12(1)));
+			FAIL_IF(push_inst(compiler, SLTUI | RD(dst_r) | RJ(OTHER_FLAG) | IMM_I12(1)));
```

Comment:
Note: I changed it for convenience, you don't need to change it if using the equal flag is easier for the target cpu.

Reply:
Either way should be okay, but I think it's better to stay the same with other RISC architectures for maintainability.